composites/OnboardingWizard/onboarding-wizard.scss

```diff
@@ -44,6 +44,23 @@ body {
   box-sizing: border-box;
   width: 100%;
 
+  &--header {
+    text-align: center;
+
+    h1 {
+      color: $color_pink_dark;
+      margin: 0 0 -16px;
+      font-size: 1.2em;
+      letter-spacing: 0.03em;
+      line-height: 2.6rem;
+      font-weight: 100;
```

Comment:
using a number as value for font-weight makes sense only when there's an actual font with that weight. The actual value computed here is "normal".